web/portal/client/cypress/e2e/CallForwardSetting/CallForwardSetting.tests.js

```diff
@@ -82,3 +82,44 @@ export const deleteCallForwardSetting = () => {
     .its('response.statusCode')
     .should('eq', 204);
 };
+
+export const postCallForwardSettingToFriends = () => {
+  cy.usePactIntercept(
+    {
+      method: 'POST',
+      url: '**/api/client/call_forward_settings*',
+      response: newCallForwardSetting.response,
+      matchingRules: newCallForwardSetting.matchingRules,
+    },
+    'createCallForwardSettingToFriends'
+  );
+
+  cy.get('[aria-label=Add]').click();
+
+  const {
+    callTypeFilter,
+    callForwardType,
+    targetType,
+    numberValue,
+    friend,
+    extension,
+    voicemail,
+    enabled,
+  } = newCallForwardSetting.request;
+  cy.fillTheForm({
+    callTypeFilter,
+    callForwardType,
+    targetType,
+    numberValue,
+    friend,
+    extension,
+    voicemail,
+    enabled,
+  });
+
+  cy.get('header').should('contain', 'Call forward settings');
+
+  cy.usePactWait('createCallForwardSettingToFriends')
+    .its('response.statusCode')
+    .should('eq', 201);
+};
```

Comment:
Do not create this, use the existent one in the top of the file, actually is ignoring tons of attributes from newItem